apps/aws-amplify/src/locations/__tests__/ConfigScreen.spec.tsx

```diff
@@ -71,25 +72,34 @@ describe('ConfigScreen', () => {
         amplifyWebhookUrl: testUrl,
       });
 
-      fireEvent.change(screen.getByTestId('webhookUrl'), {
-        target: {
-          value: testUrl,
-        },
-      });
+      fireEvent.change(
+        screen.getByPlaceholderText('ex. https://webhooks.amplify.us-east-1.amazonaws.com/...'),
+        {
+          target: {
+            value: testUrl,
+          },
+        }
+      );
     });
 
     it('overrides the previously saved parameters', async () => {
       const newUrl = 'https://www.newurl.com/';
-      const webhookUrlInput = screen.getByTestId('webhookUrl') as HTMLInputElement;
+      const webhookUrlInput = screen.getByPlaceholderText(
+        'ex. https://webhooks.amplify.us-east-1.amazonaws.com/...'
+      ) as HTMLInputElement;
 
       expect(webhookUrlInput.value).toBe(testUrl);
+      console.log('screen', screen.debug());
```

Comment:
Is this console.log meant to stay in the test?

Reply:
no, I will remove